spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/AdminController.java

```diff
@@ -176,7 +168,7 @@ public ModelAndView goProfiles(Model model) throws IOException {
     }
 
     @RequestMapping(value = "", method = GET)
-    public ModelAndView home(Model model, HttpServletRequest request) throws IOException {
+    public ModelAndView home(Model model) throws IOException {
```

Comment:
I have not idea what this method is doing by reading its name. Could you please rename it?